unity-renderer/Assets/UIComponents/Tests/ToggleComponentViewTests.cs

```diff
@@ -0,0 +1,94 @@
+using NUnit.Framework;
+using UnityEngine;
+
+public class ToggleComponentViewTests
+{
+    private ToggleComponentView toggleComponent;
+
+    [SetUp]
+    public void SetUp()
+    {
+        toggleComponent = BaseComponentView.Create<ToggleComponentView>("Toggle_Capsule");
+    }
+
+    [TearDown]
+    public void TearDown()
+    {
+        toggleComponent.Dispose();
+        GameObject.Destroy(toggleComponent.gameObject);
```

Comment:
This `Destroy` is not needed because it is already added into the `Dispose` in the `BaseComponentView` class.